### PR TITLE
docs: Fixing examples / themes

### PR DESCRIPTION
**Pull request by Scooter1337**
Closes: #701 

Fixing /examples

- Found that Model selection had a small error. (see first commit, and fourth commit for edit)

### CODE REVIEW

**Comment by Scooter1337** [comment]
- Small error
- second commit closes #701

File: apps/www/src/routes/examples/playground/(components)/model-selector.svelte [hunk 1]
```diff
@@ -106,7 +106,7 @@
 											}}
 										>
 											<Command.Item
-												value={model.name}
+												value={model.id}
```
Comment (huntabyte) [inline apps/www/src/routes/examples/playground/(components)/model-selector.svelte:109]: This breaks the ability to search, try typing in `text` into the searchbar and you'll see.

```suggestion
												value={model.name}
```

What we should do instead to enable models to be selected is to update the `onSelect` callback to:


```ts
onSelect(() => {
	value = model.id
	closeAndFocusTrigger(ids.trigger)
})
```

File: apps/www/src/lib/registry/new-york/example/cards/report-issue.svelte [hunk 2]
```diff
@@ -73,8 +73,8 @@
 			<div class="grid gap-2">
 				<Label for="security-level">Security Level</Label>
 				<Select.Root selected={securityLevels[1]}>
-					<Select.Trigger id="security-level">
-						<Select.Value placeholder="Select level" />
+					<Select.Trigger>
+						<Select.Value placeholder="Select level" id="area" />
```
Comment (huntabyte) [inline apps/www/src/lib/registry/new-york/example/cards/report-issue.svelte:77]:
```suggestion
						<Select.Value placeholder="Select level" />
```

The `value` shouldn't get an id.

Comment (Scooter1337) [reply apps/www/src/lib/registry/new-york/example/cards/report-issue.svelte:77]: I agree, but not adding an id yields a browser issue (for without id)

File: apps/www/src/routes/examples/playground/(components)/model-selector.svelte (hunk 1 above)
Comment (Scooter1337) [reply apps/www/src/routes/examples/playground/(components)/model-selector.svelte:109]: Done, thanks for keeping an eye out. Will be more careful in the future.

**Review by Scooter1337 — COMMENTED** [review]
Implements Huntabyte's recommendation

**Comment by Scooter1337** [comment]
Now closes #701

**Comment by Scooter1337** [comment]
Sadly don't really understand what's going on with the hover card issue (#697).

But this pull request still cleans up the examples!

**Comment by huntabyte** [comment]
Closing in favor of #703 which includes your commits/contributions. In the future, please submit PRs from a branch other than your `main` branch of the fork, as `main` doesn't allow me to make modifications locally and push them to the PR without jumping through hoops.

**Pull request closed without merging**